content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -160,9 +160,11 @@ You can use `NotIn` and `DoesNotExist` to achieve node anti-affinity behavior, o
 If you specify both `nodeSelector` and `nodeAffinity`, *both* must be satisfied for the pod
 to be scheduled onto a candidate node.
 
-If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types, then the pod can be scheduled onto a node **only if all** `nodeSelectorTerms` can be satisfied.
+If you specify list of `matchExpression and matchField` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node **if one of** the `matchExpression and matchField` is satisfied.
```

Comment:
```suggestion
If you specify a list of `matchExpression` and `matchField` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node if  `matchExpression` or `matchField` is satisfied.
```

Reply:
Thx for the review.
I have some different voice about this <"if  `matchExpression` or `matchField` is satisfied. ">
In the code we can see if user use <`matchExpression` and `matchField`> in one nodeSelectorTerm. Both matchExpression and matchField should be satisfied in this nodeSelectorTerm.
So maybe it can change to this blow:

If you specify a list of `matchExpression` and `matchField` associated with `nodeSelectorTerms`, then the pod can be scheduled onto a node if  one of `matchExpression`  and `matchField` is satisfied.